genie-web/src/main/java/com/netflix/genie/web/data/entities/JobEntity.java

```diff
@@ -461,6 +462,9 @@
     )
     private Set<TagEntity> tags = new HashSet<>();
 
+    @Transient
+    private JobStatus notifiedJobStatus;
```

Comment:
I'm only on commit one but this state will be lost once the entity goes out of scope if i'm not mistaken.

e.g. if we save the entity and it's running then our save API is done this field will be null again when we reload the entity.

Reply:
Correct, but i'm intercepting both `update` and `load` entity lifecycle for this exact purpose.